src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntegerTests.cs

```diff
@@ -3507,20 +3507,117 @@ static void verify(CSharpCompilation comp)
         public void AliasName_02()
         {
             var source =
-@"using N = nint;
+@"using A1 = nint;
+using A2 = nuint;
 class Program
 {
-    N F() => default;
+    A1 F1() => default;
+    A2 F2() => default;
 }";
+            var expectedDiagnostics = new[]
+            {
+                // (1,12): error CS0246: The type or namespace name 'nint' could not be found (are you missing a using directive or an assembly reference?)
```

Comment:
>// (1,12): error CS0246: The type or namespace name 'nint' could not be found (are you missing a using directive or an assembly reference?) [](start = 16, length = 139)

Please add a test for a success case, i.e. when the type or namespace can be found.